views/chat.js

```diff
@@ -0,0 +1,174 @@
+import * as database from './../services/database.js';
+
+let Chat = {
+    render: async () => {
+        return `
+        <body>
+            <header>
+                <a href="/#/chat" id="logo">Chat</a>
+                <nav>
+                    <a href="#"><i class="far fa-moon"></i></a>
+                    <a href="#/add-chat"><i class="fas fa-plus-circle"></i></a>
+                    <a href="#/settings"><i class="fas fa-user-cog"></i></a>
+                    <a href="#" id="logout-btn"><i class="fas fa-sign-out-alt"></i></a>
```

Comment:
`a` without a valid `href` should be replaced with buttons.

Reply:
Fixed